packages/sst/src/constructs/Function.ts

```diff
@@ -534,6 +539,50 @@ export interface PythonProps {
   installCommands?: string[];
 }
 
+/**
+ * Used to configure Go bundling options
+ */
+export interface GoProps {
+  /**
+   * The ldflags to use when building the Go module.
+   *
+   * @default ["-s", "-w"]
+   * @example
+   * ```js
+   * go: {
+   *   ldFlags: ["-X main.version=1.0.0"],
+   * }
+   * ```
+   */
+  ldFlags?: string[];
+
+  /**
+   * The build tags to use when building the Go module.
+   *
+   * @default ""
+   * @example
+   * ```js
+   * go: {
+   *   buildTags: "enterprise pro",
+   * }
+   * ```
+   */
+  buildTags?: string;
```

Comment:
Should this take a list of string? ie. `["enterprise", "pro"]`

Reply:
Good catch! Go uses []string internally https://pkg.go.dev/go/build#Context.